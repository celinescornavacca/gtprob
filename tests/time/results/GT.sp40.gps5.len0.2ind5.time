3.95user 0.15system 0:04.27elapsed 96%CPU (0avgtext+0avgdata 64068maxresident)k
140inputs+0outputs (0major+32650minor)pagefaults 0swaps
