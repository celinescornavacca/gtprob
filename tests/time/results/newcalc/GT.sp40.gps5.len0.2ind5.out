-385.020958281
